tests/default.nix

```
@@ -142,6 +142,7 @@ let
       "nix-index"
       "nix-your-shell"
       "ollama"
+      "onlyoffice"
```

```suggestion
      "onlyoffice-desktopeditors"
```

This is the actual package name from nixpkgs.

---

```bash
nix build .#onlyoffice                                                                              
error: flake 'git+file:///home/khaneliman/Documents/github/nixpkgs' does not provide attribute 'packages.x86_64-linux.onlyoffice', 'legacyPackages.x86_64-linux.onlyoffice' or 'onlyoffice'
```
```bash
nix build .#onlyoffice-desktopeditors
these 3 derivations will be built:
  /nix/store/m7vv4bklz3hc532yv33ykxy68fpjs7g3-onlyoffice-desktopeditors-8.3.1-init.drv
  /nix/store/8bsc57m1n87r85h5dn581k9z5b9gxsbg-onlyoffice-desktopeditors-8.3.1-bwrap.drv
  /nix/store/4v2rn0h9jjvac19mv7m3zzxqn91i5f9d-onlyoffice-desktopeditors-8.3.1.drv